app/editor/src/components/form/text/FormSizing.ts

```diff
@@ -0,0 +1,7 @@
+export enum FormInputSizes {
```

Comment:
LOL I did a similar thing as you, but didn't use class names.  I'm wondering which makes the most sense?  Checkout out my `FieldSize` enum.

Reply:
I saw that haha, I like your better I think...